src/instructlab/model/chat.py

```diff
@@ -585,6 +587,32 @@ def _handle_list_contexts(self, _):
         self._sys_print(Markdown(f"**Available contexts:**\n\n{context_list}"))
         raise KeyboardInterrupt
 
+    def _handle_set_custom_context(self, content):
+        parts = content.split(" ", 2)
+        if len(parts) < 3:
+            self._sys_print(
+                Markdown(
+                    "**ERROR**: Please use `/sc context_name custom_context_text`. "
+                    "Note: Use underscores (`_`) to join multiple words in `context_name`."
+                )
+            )
+            raise KeyboardInterrupt
+
+        context_name = parts[1]
+        custom_context_text = parts[2]
+
+        CONTEXTS[context_name] = custom_context_text
```

Comment:
I'd rather not allow to override default contexts. Yes, you can restart chat session, but it would be better if the user can always reset to default, whatever they do.